cub/agent/agent_radix_sort_onesweep.cuh

```diff
@@ -219,11 +211,8 @@ struct AgentRadixSortOnesweep
                 // write the local sum into the bin
                 AtomicOffsetT& loc = d_lookback[block_idx * RADIX_DIGITS + bin];
                 OffsetT value = bins[u] | LOOKBACK_PARTIAL_MASK;
-#if defined(__NVCOMPILER_CUDA__)
-                (volatile OffsetT&)loc = value;
-#else
-                loc.store(value, cuda::memory_order_relaxed);
-#endif
+                //(volatile OffsetT&)loc = value;
+                ThreadStore<STORE_CG>(&loc, value);
```

Comment:
Why not `STORE_VOLATILE` to match the previous behavior? Was volatile too strong here?